linux/home/.tmux.conf

```diff
@@ -0,0 +1,25 @@
+# TODO: double check where zsh is installed
+# set-option -g default-shell /usr/bin/zsh
```

Comment:
The easiest way to maybe implement this is inside `bootstrap.sh` with a prepend after double checking zsh install location with `which`. On failure fallback to `bash`